src/app/shared/versions.ts

```diff
@@ -28,6 +28,7 @@ import { Tooltip } from './tooltip';
 export abstract class Versions extends EntryTab {
   @Input() versions: Array<Tag | WorkflowVersion>;
   @Input() verifiedSource: Array<any>;
+  @Input() isTool: boolean;
```

Comment:
Is this even used?

Reply:
it was being used after all in is container.component.html

Reply:
I see this PR passing in this new `isTool`, but I don't see it used in the .ts or .html

Reply:
it's on line 226 in container.component.html

Reply:
Yes, you added it in the versions.ts and container.component.html, but where is it used?

Reply:
Ohhh sorry Gary.